inference/core/workflows/core_steps/models/roboflow/object_detection.py

```diff
@@ -223,11 +268,11 @@ async def run_remotely(
         max_candidates: Optional[int],
         disable_active_learning: Optional[bool],
         active_learning_target_dataset: Optional[str],
-    ) -> List[Dict[str, Union[sv.Detections, Any]]]:
+    ) -> BlockResult:
         api_url = (
             LOCAL_INFERENCE_API_URL
             if WORKFLOWS_REMOTE_API_TARGET != "hosted"
-            else HOSTED_CLASSIFICATION_URL
+            else HOSTED_DETECT_URL
```

Comment:
wut?  this one was definitely working before right?  does it work with the classification url also and we have the different url just in case it changes?

Reply:
yeah, it makes no difference almost, but prefer to bring it to order